.mypy.ini

```diff
@@ -1,8 +1,13 @@
 [mypy]
-python_version = 3.7
+python_version = 3.11
+enable_error_code = ignore-without-code
 ignore_missing_imports = True
 follow_imports_for_stubs = True
+no_implicit_optional = True
+show_column_numbers = True
 warn_redundant_casts = True
+; warn_return_any = True  ; TODO: Enable this check
```

Comment:
This is a valuable rule to have however when enabling it we go from 54 errors to 78.

So I left this as something we can hopefully enable at some point.